BNFC-meta/BNFC-meta.cabal

```
@@ -70,3 +70,29 @@ Library
     , Language.LBNF.TypeChecker
     , Language.LBNF.Utils
     , Language.Haskell.TH.Hide
+
+test-suite tests
+  type:                exitcode-stdio-1.0
+  main-is:             Tests.hs
+  hs-source-dirs:      test
+  other-modules:       Grammars.Arithmetic
+                     , Grammars.CMinusMinus
+                     , Grammars.JSON
+                     , Grammars.LabelledBNF
+                     , Grammars.Lambda
+                     , Grammars.GenCode
+                     , Instances
+  build-depends:       base < 5
+                     , tasty >= 1.5 && < 1.6
+                     , tasty-hunit >= 0.10 && < 0.11
+                     , tasty-quickcheck >= 0.11 && < 0.12
+                     , QuickCheck >= 2.14 && < 2.16
+                     , generic-arbitrary >= 1.0 && < 1.1
+                     , BNFC-meta >= 0.6 && < 0.7
```

```suggestion
                     , BNFC-meta
```
Dependency on a library inside the same package is usually not versioned.